src/options.js

```diff
@@ -11,8 +13,16 @@ var defaults = {
   bracketSpacing: true
 };
 
+var exampleConfig = Object.assign({}, defaults, {
+  filename: "testFilename",
+  useFlowParser: false,
+  printWidth: 80,
+  originalText: "text"
+});
+
 // Copy options and fill in default values.
 exports.normalize = function(options) {
+  validate(options, { exampleConfig });
```

Comment:
I've used it like this because `options` contain some extra keys, which are not present in `defaults`.